src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -87,7 +121,65 @@ def ramp_fit(model, buffsize, save_opt, readnoise_2d, gain_2d,
         Object containing optional GLS-specific ramp fitting data for the
         exposure
     """
+    ramp_class = create_ramp_fit_class(model, dqflags)
```

Comment:
Perhaps a comment stating that this is an instance of an internal ramp class.

Reply:
I added a comment to clarify a class is being created using only the needed information from the data model.